aviary/mission/flops_based/phases/energy_phase.py

```diff
@@ -368,6 +370,15 @@ def _extra_ode_init_kwargs(self):
         }
 
 
+EnergyPhase._add_meta_data(
+    'reserve', val=False, desc='this phase is part of the reserve mission.')
+
+EnergyPhase._add_meta_data(
+    'target_distance', val={}, desc='the amount of distance traveled in this phase added as a constraint')
+
+EnergyPhase._add_meta_data(
+    'target_time', val={}, desc='the amount of time taken by this phase added as a constraint')
```

Comment:
I wonder if "target_duration" is a better name for this, given that we call it duration in the dymos options, and have adopted that in aviary as well.

Reply:
After a long conversation with Ken, 
target_duration has been implemented instead of target_time.